test/unit/query/builder.js

```diff
@@ -40,6 +40,32 @@ const clientsWithNullAsDefault = {
   ),
 };
 
+var customLoggerConfig = {
```

Comment:
would be better to convert all vars in this file to let/const

Reply:
changed to const